apps/modernization-ui/src/apps/patient/data/address/AddressEntryFields.spec.tsx

```diff
@@ -133,4 +133,23 @@ describe('AddressEntryFields', () => {
             expect(queryByText('Use is required.')).not.toBeInTheDocument();
         });
     });
+
+    it('should invalidate Census Tract format', async () => {
+        const { getByLabelText, getByText } = render(<Fixture />);
+
+        const censusTractInput = getByLabelText('Census tract');
+
+        act(() => {
+            userEvent.clear(censusTractInput);
+            userEvent.type(censusTractInput, '12345');
+            userEvent.tab();
+        });
```

Comment:
Question: What are your thoughts on adding some additional test cases?

|Value|Valid|
|----|----|
|0000.00|false|
|0001.00|false|
|0001.01|true|
|1000.00|false|
|1000.01|true|
|9999.99|false|
|9999.98|true|

Reply:
I have added most of these scenarios but the last two from this list as the user would not be permitted to enter more than 4 digits, similarly with the decimals it would not exceed the 2 decimals, the others were helpful though so thanks